src/main/webapp/web.xml

```diff
@@ -105,4 +136,14 @@
         <servlet-name>deleteManufacturer</servlet-name>
         <url-pattern>/manufacturers/delete</url-pattern>
     </servlet-mapping>
+
+    <filter>
+        <filter-name>authenticationFilter</filter-name>
+        <filter-class>taxi.web.filter.AuthenticationFilter</filter-class>
+    </filter>
+    <filter-mapping>
+        <filter-name>authenticationFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
```

Comment:
It's fine to use mapping in web.xml, but short tip for you- we can use @WebServlet("/servletPath")  and @WebFilter(urlPatterns = "/pagesPathThatWeNeedToFilter") over servlet and filter classes. Just shared my experience, it makes working with servlets realy faster, and your code much readable, even if you need to check what path does your servlet have

Reply:
I do this in previous task but here we have already have this mapping in web.xml so I decide to use an other solution as we have implemented here ...